Documentation/installation/k8s-install-migration.rst

```diff
@@ -193,6 +193,34 @@ Preparation
         $ cilium install --helm-values values-migration.yaml --helm-auto-gen-values values-initial.yaml
         $ cat values-initial.yaml
 
+.. note::
+In case you are migrating from Calico you need to ensure that Calico doesn't accidently switch Cilium's interfaces during the migration.
```

Comment:
```suggestion
When you install Cilium, it will create a new network interface called ``cilium_host`` on each node. You may need to configure the old CNI to ensure that it doesn't accidentally use this interface.
```
Moved a sentence from below to here, I think it's a clearer way of setting the scene. The same problem is very likely to be true for other CNIs, so I suggest we use Calico here as an example (see also next suggestion)

Reply:
+1 to @lizrice's change. The only thing I would add is a verb tense nit: "it will create" to "it creates".